tests/unit/array/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED: 'true'
       PERSISTENCE_DATA_PATH: '/var/lib/weaviate'
   qdrant:
-    image: qdrant/qdrant:v0.7.0
+    image: qdrant/qdrant:v0.10.0
```

Comment:
Are we sure we want to go all they way up to v10? I vaguely remember that there was some issue with qdrant 10, have we resolved that? @JoanFM

Reply:
Yes, with this, we can bump our client to version 8.